wix/Build.OpenJDK_generic.cmd

```diff
@@ -70,6 +70,10 @@ IF NOT "%ARCH%" == "x64" (
 	)
 )
 
+REM Update to handle the change of build variant until implications
+REM of setting this to Temurin can be evaluated
+IF "%JVM%" == "temurin" SET JVM=hotspot
```

Comment:
@sxa, @karianna  Why this special case is not handled on the caller side ?
temurin is not a JVM so the JVM arg must not accept "temurin" and hide some other value behind ..

If you set upsteam a vendor name temurin , his place is in VENDOR_BRANDING from this script, not in JVM arg 

I think in caller job you must do somthing like that: 
if (VARIANT == "temurin" )
 VENDOR_BRANDING"=Temurin"
 JVM="hotspot" // default JVM for VARIANT "temurin"
 call createInstallerJob
fi

Reply:
The intention of the initial set of changes was to do the minimum required in the pipelines to enable the use of a temurin variant. There will likely be subsequent changes to split this out using other variables but initially this was the simplest option to prove that we could split out hotspot and temurin. After implementing it, we hit this issue and this was the simplest way to fix it which wouldn't involve any risk to changing the contents of the package.

I agree this could be switched to being done in the calling pipelines, and so your proposed change can be covered in the other issue which Martijn has created :-)